content/logs/languages/php.md

```diff
@@ -0,0 +1,503 @@
+---
+title: PHP log collection
+kind: documentation
+further_reading:
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+- link: "/logs/explore"
+  tag: "Documentation"
+  text: Learn how to explore your logs
+- link: /logs/faq/log-collection-troubleshooting-guide
+  tag: "FAQ"
+  text: Log Collection Troubleshooting Guide
+---
+
+## Overview
+
+Write your PHP logs into a file, then [use the Agent](/logs) to forward them to Datadog, choose between the following libraries:
```

Comment:
Should this `[use the Agent](/logs)` link to `/agent` instead?

> `[use the Agent](/agent)`

Reply:
Nop because agent configuration instructions for log collection are in `/logs`

Reply:
Ahh, gotcha - thanks Gus! :)